src/selectors/resourceSelectors.js

```diff
@@ -0,0 +1,101 @@
+import { createSelector } from 'reselect';
+
+const getAllResources = state => state.filterMarkers.allResources;
+const getResourceType = state => state.filterMarkers.resourceType;
+const filterTags = state => state.filterMarkers.filterTags;
+const filterEntry = state => state.filterMarkers.filterEntry;
+
+const tagMapping = {
+  //ENTRY TYPES
+  'Open Access': 'OPEN',
+  'Restricted': 'RESTRICTED',
+  'Unsure': 'UNSURE',
+  //WATER
+  //dispenser types
+  'Drinking fountain': 'DRINKING_FOUNTAIN',
+  'Bottle filler': 'BOTTLE_FILLER',
+  'Sink': 'SINK',
+  'Water cooler': 'WATER_COOLER',
+  'Soda machine': 'SODA_MACHINE',
+  'Vessel': 'VESSEL',
+  //features
+  'ADA accessible': 'WHEELCHAIR_ACCESSIBLE',
+  'Filtered water': 'FILTERED',
+  'Vessel needed': 'BYOB',
+  'ID required': 'ID_REQUIRED',
+  //FOOD
+  //food types
+  'Perishable': 'PERISHABLE',
+  'Non-perishable': 'NON_PERISHABLE',
+  'Prepared foods and meals': 'PREPARED',
+  //distribution types
+  'Eat on site': 'EAT_ON_SITE',
+  'Delivery': 'DELIVERY',
+  'Pick up': 'PICKUP',
+  //organization types
+  'Government': 'GOVERNMENT', 'Business': 'BUSINESS', 'Non-profit': 'NON_PROFIT',
+  //FORAGING
+  //forage type
+  'Nut': 'NUT', 'Fruit': 'FRUIT', 'Leaves': 'LEAVES', 'Bark': 'BARK', 'Flowers': 'FLOWERS',
+  //features
+  'Medicinal': 'MEDICINAL', 'In season': 'IN_SEASON', 'Community garden': 'COMMUNITY_GARDEN',
+  //BATHROOM
+  //features
+  'Gender neutral': 'GENDER_NEUTRAL',
+  'Changing table': 'CHANGING_TABLE',
+  'Single occupancy': 'SINGLE_OCCUPANCY',
+  'Family bathroom': 'FAMILY',
+  'Has water fountain': 'HAS_FOUNTAIN',
+}
+
+const propertyMapping = {
+  'WATER': ['dispenser_type', 'tags'],
+  'FOOD': [],
+  'FORAGE': [],
```

Comment:
Include 'tags' in these as well

Reply:
Sorry that was a lie - they have these types: https://github.com/phlask/phlask-map/blob/develop/src/types/ResourceEntry.js#L74-L89

Reply:
fixed